Connection-Issues.md

```diff
@@ -0,0 +1,27 @@
+# Connection Issues
+
+In rare occasions, Internet Service Providers have issues on their end preventing users from connecting to [Runelite's website](https://runelite.net/), which is a requirement for starting Runelite. While this problem is not the fault of the user, it can often be mitigated with one of the workarounds listed below.
+
+#### Unresolved issues:
+`Globe` in The Philippines has had this issue since 2023-12-03
```

Comment:
nit-pick: code styling (`Globe`, in this case) doesn't really make sense here. *Italics* or **bold** would probably be better.